Source/JavaScriptCore/bytecode/InlineAccess.h

```diff
@@ -56,6 +56,8 @@ class InlineAccess {
 #else
         return 52;
 #endif
+#elif CPU(MIPS)
+        return 48;
```

Comment:
array length size: 56
out of line offset cache size: 72
inline offset cache size: 68
replace cache size: 68
replace out of line cache size: 72